### PR TITLE
Automated cherry pick of #94985: Fix `kubectl describe ingress` format

### PR DESCRIPTION
**Pull request by soltysh**
Cherry pick of #94985 on release-1.19.

#94985: Fix `kubectl describe ingress` format

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

### CODE REVIEW

**Comment by soltysh** [comment]
/assign @liggitt @seans3 
@kubernetes/release-managers for cherry-pick approval

File: staging/src/k8s.io/kubectl/pkg/describe/describe_test.go [hunk 1]
```diff
@@ -1924,6 +1924,28 @@ Rules:
...
+					Name:      "bar",
+					Namespace: "foo",
+				},
+				Spec: networkingv1.IngressSpec{
+					DefaultBackend: &backendV1,
+				},
+			}),
+			output: `Name:             bar
+Namespace:        foo
+Address:
```
Comment (rikatz) [inline staging/src/k8s.io/kubectl/pkg/describe/describe_test.go:1939]: Apparently it's missing 10 spaces here, like: https://github.com/kubernetes/kubernetes/blob/b895f8c7a697a97acf28067b2f38a0dc7d822c02/staging/src/k8s.io/kubectl/pkg/describe/describe_test.go#L1770

Comment (soltysh) [reply staging/src/k8s.io/kubectl/pkg/describe/describe_test.go:1939]: Right, good catch.

**Comment by rikatz** [comment]
Tested, everything fine now :)

/lgtm

Thank you!

**Comment by rikatz** [comment]
/test pull-kubernetes-node-e2e
flakes :(

**Comment by rikatz** [comment]
ping @kubernetes/release-managers

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**